### PR TITLE
Add documentation about the Gravatar OAuth

### PR DESCRIPTION
**Pull request by pinarol**
Closes #

### Description

Adds documentation about the Gravatar OAuth.

### Testing Steps

Build the docs and review.

### CODE REVIEW

File: Sources/GravatarUI/GravatarUI.docc/GravatarOAuth.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            redirectURI: "https://yourhost.com/redirect-url"
+        )
+    )
+}
+```
+
+3. Define a Universal Link for the redirectURI
+
+Please ensure that a Universal Link is defined in your application for the redirect URI, so the OAuth flow can run successfully. For the required steps, please refer to the [Universal Links](https://developer.apple.com/documentation/xcode/allowing-apps-and-websites-to-link-to-your-content/) documentation.
+
```
Comment (etoledom) [inline Sources/GravatarUI/GravatarUI.docc/GravatarOAuth.md]: Looking great. Just one small last detail:
According to [this guideline from Google](https://developers.google.com/style/tone#politeness-and-use-of-please) (which I find pretty useful) is not recommended to over-use the word 'please'.

```suggestion
For the OAuth flow to run successfully, ensure that a Universal Link is defined in your application for the redirect URI. For the required steps, refer to the [Universal Links](https://developer.apple.com/documentation/xcode/allowing-apps-and-websites-to-link-to-your-content/) documentation.

```

We don't strictly follow this guideline though, so you can decide. As it is now, is good already 👍

**Review by etoledom — APPROVED** [review]
Nice one! Thank you for this.

I added a small suggestion, but feel free to merge directly.

File: Sources/GravatarUI/GravatarUI.docc/GravatarOAuth.md (hunk 1 above)
Comment (pinarol) [reply Sources/GravatarUI/GravatarUI.docc/GravatarOAuth.md]: I like it. 👍

Comment (andrewdmontgomery) [reply Sources/GravatarUI/GravatarUI.docc/GravatarOAuth.md]: I love this guideline.  Now I'm wondering how often I use `please` in documentation.  I don't _think_ I do...

**Pull request merged**